src/org/zaproxy/zap/extension/fuzz/resources/Messages.properties

```diff
@@ -219,16 +219,20 @@ fuzz.payloads.generator.script.warnNoNumberOfpayloads.title = Unknown Number Of
 
 fuzz.payloads.generator.regex.name = Regex (*Experimental*)
 fuzz.payloads.generator.regex.regex.label = Regex:
-fuzz.payloads.generator.regex.description.base = {0} (Limited to {1} payloads{2})
-fuzz.payloads.generator.regex.description.length = \ and maximum length of {0} characters
+fuzz.payloads.generator.regex.description.base = {0} {1}
+fuzz.payloads.generator.regex.description.maxPayloads = (Limited to {0} payloads)
 fuzz.payloads.generator.regex.maxPayloads.label = Max. Payloads:
-fuzz.payloads.generator.regex.maxPayloads.tooltip = <html>The maximum number of payloads that should be generated by the regular expression.<br>Only meaningful when using quantifiers '*' and '+'.</html>
-fuzz.payloads.generator.regex.maxLength = Max. Length:
-fuzz.payloads.generator.regex.maxLength.tooltip = <html>The maximum length of the payloads that should be generated by the regular expression.<br>Only meaningul when using quantifiers '*' and '+'.<br>Value zero means no maximum length thus restricted by maximum number of payloads</html>
+fuzz.payloads.generator.regex.maxPayloads.tooltip = The maximum number of payloads that should be generated by the regular expression (zero for no limit).
+fuzz.payloads.generator.regex.warnInfiniteRegex.message = The specified regular expression generates an infinite, or very large, number of strings.\r\nContinue anyway?
+fuzz.payloads.generator.regex.warnInfiniteRegex.title = Infinite Regular Expression
+fuzz.payloads.generator.regex.warnInvalidRegex.message = The specified regular expression is not syntactically valid.
```

Comment:
You may want to expand on "syntactically".  That may not translate well and is not a term that is used often.